packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -208,6 +209,16 @@ export default class GoogleCalendarService implements Calendar {
       payload["location"] = getLocation(calEventRaw);
     }
 
+    if (calEventRaw.recurringEvent) {
+      const rule = new RRule({
+        freq: calEventRaw.recurringEvent.freq,
+        interval: calEventRaw.recurringEvent.interval,
+        count: calEventRaw.recurringEvent.count,
+      });
+
+      payload["recurrence"] = [rule.toString()];
+    }
```

Comment:
I am not sure if every calendar follows a different format for recurring event creation, but if it doesn't we should really move this code to core so that it's available consistently to all calendars.